cli/src/main.rs

```diff
@@ -476,10 +479,45 @@ async fn main() -> Result<()> {
                 }
             }
         }
-        Command::MarkBackendLost { cluster, backends } => {
+        Command::MarkBackendLost {
+            cluster,
+            backends,
+            force,
+        } => {
+            let stdin = std::io::stdin();
+            let state = get_world_state(nats.clone()).await?;
             let cluster = ClusterName::new(&cluster);
+            let cluster_state = state
+                .cluster(&cluster)
+                .ok_or_else(|| anyhow::anyhow!("{cluster} cluster does not exist"))?;
             for backend in backends {
                 let backend = BackendId::new(backend);
+                let Some(backend_state) = cluster_state.backend(&backend) else {
+                    println!("{} no such backend!", backend.id());
+                    continue;
+                };
+                println!("{backend_state:?}");
+                if backend_state
+                    .state()
+                    .map_or(false, |state| state.terminal())
+                {
+                    println!(
+                        "backend state {} is a terminal state!",
+                        backend_state.state().unwrap()
+                    );
+                }
```

Comment:
If the backend is already in a terminal state, then I think we should `continue` and not allow pushing a Lost state to it. The constraint that a backend can only have one terminal state in its history of states is a useful one and is trivial to enforce here.